### ISSUE TITLE
Add IOS appear animation marker 

### ISSUE THREAD

**Issue by yedeba** [wf9a5m75/phonegap-googlemaps-plugin#276]
Hi,
This is my implementation for a marker appears on the map with animation, works great, just what I've done for iOS, does anyone know how to do it for Android?

File googlemaps-cdv-plugin.js:

Line 764 Add
```
markerOptions.appearanimation = markerOptions.appearanimation || false;
```

Line 1154 Add
```
Marker.prototype.setAppearAnimation = function(appearanimation) {
  this.set('appearanimation', appearanimation);
  cordova.exec(null, this.errorHandler, PLUGIN_NAME, 'exec', ['Marker.setAppearAnimation', this.getId(), appearanimation]);
};
```

File marker.m:
Line 32 Add
```
if ([[json valueForKey:@"appearanimation"] boolValue] == true) {
    marker.appearAnimation = kGMSMarkerAnimationPop;
 }
```
Example of use:
```
map.addMarker({
	'position': latlon,
	'icon': icon,
	'appearanimation': true,
}
```

**Comment by wf9a5m75** [comment]
Since there is no way to popup animation for Android, I won't implement this feature.
However the marker drop animation(#208) will be implemented instead. (coming soon)

**Comment by wf9a5m75** [comment]
I implemented the marker animation feature (#208).
